devsetup/scripts/crc-setup.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# set -x
+set -x
```

Comment:
[creds](https://github.com/openstack-k8s-operators/install_yamls/pull/965/files#diff-5fe68a3a8ef2686186052e6114a3d72939895df06382cc44ce9bb24ce4feeca4R70) and KUBEADMIN_PWD should not be echoed